src/OrchardCore/OrchardCore.ContentManagement.Display/ContentItemDisplayManager.cs

```diff
@@ -64,10 +64,17 @@ public async Task<IShape> BuildDisplayAsync(ContentItem contentItem, IUpdateMode
             }
 
             var stereotype = contentTypeDefinition.GetSettings<ContentTypeSettings>().Stereotype;
-            var actualDisplayType = string.IsNullOrEmpty(displayType) ? "Detail" : displayType;
-            var actualShapeType = stereotype ?? "Content";
+            var actualDisplayType = String.IsNullOrEmpty(displayType) ? "Detail" : displayType;
```

Comment:
i guess we keep `string`

Reply:
Based on the suggestion in the [.editorconfig](https://github.com/OrchardCMS/OrchardCore/blob/ff41ba1da36538bbd2a5af036657c8f7b68e19c1/.editorconfig#L37) it should be `String` for `string`. We use language types when defining a member. But we use Framework language when accessing  members.

Reply:
It’s unnecessary noice in PR for existing code. May be we should change editor config then

Reply:
I personally won’t want to change editor. If that is something you want, another PR can be created for that request.
I think in order to clean up the code gradually, these changes should be made to adhere to OC style suggestion otherwise what is the point of adding a suggestion.